README.md

```diff
@@ -2,51 +2,57 @@
 
 Based on [Openshift Console dynamic plugin](https://github.com/openshift/console/tree/master/frontend/packages/console-dynamic-plugin-sdk), this plugin implement the console elements for Network Observability.
 
-## Plugin development
+## Building, testing
 
-To build the plugin, use the npm command :
+To build the plugin, run:
 
+```bash
+make build
 ```
-$ npm install
-```
-
-and then :
 
-```
-$ npm build
-```
+This is equivalent to the npm commands `npm install && npm run build`.
 
-This should produce the following files :
+To run tests and linter:
 
-```
-$ tree dist/
```

Comment:
I've also removed that, it doesn't seem super useful to me but let me know if you think otherwise :-) )